homeassistant/components/geniushub/__init__.py

```diff
@@ -19,11 +27,25 @@
 )
 from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.event import async_track_time_interval
+import homeassistant.util.dt as dt_util
+
+ATTR_DURATION = "duration"
 
 _LOGGER = logging.getLogger(__name__)
 
 DOMAIN = "geniushub"
 
+# temperature is repeated here, as it gives access to high-precision temps
+GH_ZONE_ATTRS = ["mode", "temperature", "type", "occupied", "override"]
+GH_DEVICE_ATTRS = [
+    "luminance",
```

Comment:
If we make this a dict, mapping the device attribute keys to the entity state attribute keys, we don't need the `convert_dict` formatter.

Reply:
let me check if any of them are nested (after the school run)